charts/metering-ansible-operator/templates/_helpers.tpl

```diff
@@ -40,6 +40,8 @@ template:
       env:
       - name: OPERATOR_NAME
         value: "metering-ansible-operator"
+      - name: DISABLE_OCP_FEATURES
+        value: "$(DISABLE_OCP_FEATURES)"
```

Comment:
This value doesn't look right. You want `"{{ .Values.operator.disableOCPFeatures }}"` probably.